contrib/babelfishpg_common/src/collation.c

```diff
@@ -601,6 +629,33 @@ translate_collation(const char *collname, bool check_for_server_collation_name_g
 	return idx;
 }
 
+/* Translate BBF collation to it's closest SQL Server Collation. */
+char *
+translate_collation_bbf(const char *collname)
+{
+	char *res = NULL;
+	int first = 0;
+	int last = TOTAL_COLL_TRANSLATION_COUNT_TSQL - 1;
+	int middle = 16; /* optimization: usually it's the default collation. */
```

Comment:
Use TOTAL_COLL_TRANSLATION_COUNT_TSQL/2 instead of hardcoded value here

Reply:
We are choosing fast path here because we are expecting that default TSQL collation would mostly be sql_latin1_general_cp1_ci_as ~ bbf_unicode_cp1_ci_as.

Reply:
Oh I see that now, we should add proper comment in that case. Use macro in this case then